models/RequestModel.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const RequestModel = new Schema(
+  {
+    act: { type: String, required: false },
```

Comment:
act should be required, we'll set in via page always

Reply:
Made the changes.